rust/tests/namespace.ts

```diff
@@ -874,9 +1210,26 @@ describe("namespace", () => {
     }
 
     const [namespace, _namespaceBump] = await pdas.getNamespacePDA(nsMint);
-    var nsData = await namespaceProgram.fetchNamespace(namespace);
+    let nsData = await namespaceProgram.fetchNamespace(namespace);
```

Comment:
https://github.com/raindrops-protocol/raindrops/pull/28/files#r945335531

Reply:
I think this is fine how it is personally

Reply:
changed